### PR TITLE
Add axeptio, facebook & google ads tracker

### PR DESCRIPTION
**Pull request by sandrica89**

### CODE REVIEW

File: frontend/src/components/Landing/StudentLanding.jsx [hunk 1]
```diff
@@ -22,15 +22,31 @@ const StudentLanding = () => {
     if (emailRef.current) {
       emailRef.current.focus();
     }
+
+    if (__PIXEL_ADS__) {
+      fbq('track', 'PageView');
```
Comment (sandrica89) [inline frontend/src/components/Landing/StudentLanding.jsx:27]: Not sure about that, I asked Chaïb if we need to keep it.

File: frontend/index.html [hunk 2]
```diff
@@ -3,30 +3,58 @@
...
+          n.callMethod.apply(n, arguments) : n.queue.push(arguments)
+      };
+      if (!f._fbq) f._fbq = n; n.push = n; n.loaded = !0; n.version = '2.0';
+      n.queue = []; t = b.createElement(e); t.async = !0;
+      t.src = v; s = b.getElementsByTagName(e)[0];
+      s.parentNode.insertBefore(t, s)
+    }(window, document, 'script',
+      'https://connect.facebook.net/en_US/fbevents.js');
+    fbq('init', '3078857952370640');
+  </script>
```
Comment (sandrica89) [inline frontend/index.html:44]: I removed the tracking of the page view here, to add it only on student landing

**Comment by sandrica89** [comment]
I activated __PIXEL_ADS__ to test. Here are the calls made to facebook (in blue) and google (in red):
browser-network-on-email-sent

File: frontend/src/components/Landing/StudentLanding.jsx (hunk 1 above)
Comment (sandrica89) [reply frontend/src/components/Landing/StudentLanding.jsx]: Removed

**Review by desoindx — APPROVED** [review]

**Pull request merged**